apps/coordinator/src/components/Wallet/WalletGenerator.jsx

```diff
@@ -264,7 +264,11 @@ class WalletGenerator extends React.Component {
     try {
       const { getBlockchainClient } = this.props;
       const client = await getBlockchainClient();
-      await client.getFeeEstimate();
+      if (client.bitcoindParams.walletName) {
+        await client.getWalletInfo();
```

Comment:
If we have a wallet name then this is the way we need to check if the connection details are correct. Otherwise we fallback to checking fee estimate.